### PR TITLE
Add _get_figure method in plotly module

### PR DESCRIPTION
**Pull request by wizmer**
to be able to reuse the figure outside of neurom

Numpy version is constrained until the bug in pylint in conjunction
with numpy 1.16 is fixed.
PyCQA/pylint#2414

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
         'future>=0.16.0',
         'h5py>=2.7.1',
         'matplotlib>=1.3.1',
-        'numpy>=1.8.0',
+        'numpy>=1.8.0,<1.16',
```
Comment (mgeplf) [inline setup.py]: why?

Comment (wizmer) [reply setup.py]: Cf updated summary.

Comment (mgeplf) [reply setup.py]: wish it didn't have to be that way, but, lgtm

Comment (wizmer) [reply setup.py]: Hopefully it should be for a short amount of time...

**Pull request merged**